exercises/bank-account/bank_account_test.go

```diff
@@ -65,6 +65,12 @@ func TestSeqOpenBalanceClose(t *testing.T) {
 		t.Log("Balance still available on closed account.")
 		t.Fatalf("a.Balance() = %d, %t.  Want ok == false", b, ok)
 	}
+			
```

Comment:
The 'go fmt' caused the travis-ci check to fail. Simply run 'go fmt exercises/bank-account/bank_account_test.go' and push a commit for the tiny white-space change.  It should then pass travis-ci check.